pyproject.toml

```diff
@@ -17,16 +17,15 @@ classifiers = [
     "Topic :: Scientific/Engineering :: Information Analysis",
     "License :: OSI Approved :: MIT License",
     "Operating System :: OS Independent",
-    "Programming Language :: Python :: 3.8",
-    "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
 ]
 
 dependencies = [
     "ansys-api-sherlock==0.1.33",
     "grpcio>=1.17",
-    "importlib-metadata>=4.0,<5; python_version<='3.8'",
+    "importlib-metadata>=4.0,<5; python_version<='3.10'",
```

Comment:
```suggestion
```
Above 3.8 this package is included with Python so no need to define it

See https://docs.python.org/3/library/importlib.metadata.html

Reply:
I removed this line.